qna/General-firewall-validation.md

```diff
@@ -0,0 +1,16 @@
+## Question
+How can I test the connection to the fiskaltrust Services in case the Middleware has connection issues?
+
+## Metadata tags
+lang-en, market-all, middleware, PosDealer, PosOperator
+
+## Answer
+If the Middleware is showing error that hint to connection issues to one of our cloud services (e.g. "configuration cannot be downloaded" or "an error occured while uploading data"), it might be a Firewall problem. 
+Pinging our servers won't work, as they are configured to not respond to those requests - but users should try to run the following requests in Powershell:
+
+```ps
+Invoke-RestMethod 'https://helipad.fiskaltrust.cloud/version'
```

Comment:
wouldn't it be better to use /health instead of /version ?

Reply:
I thought about this, but don't think so; the /health endpoint contains much more information that is actually not relevant in this case and will just confuse the user IMO